FSW/main.cpp

```diff
@@ -104,8 +111,14 @@ int main() {
         /* - ONLY EXECTUTE DURING NORMAL OPERATION - */
         if (scienceMode.getMode() != SAFE_MODE) {
             scienceMemoryHandling();
+            handleFaults();
         }
 
+         /* - ONLY EXECUTE ON ENTRY TO STANDBY MODE - */
+         if (scienceMode.onStandbyEntry.checkInvoked()) {
+            // reset fault occurence counts
+            resetFaultCounts();
+         }
 
          /* - ONLY EXECUTE IN STANDBY MODE - */
```

Comment:
I want to say faults should be handled in standby mode as well.

Reply:
The faults are still handled in standby mode, it's just that the fault counters are reset when standby mode is entered.  Since the software passes sequentially through `standby -> sunset -> presunrise-> sunrise -> standby` once per orbit, faults will be tallied over each orbit so systemic issues will stand out. You can think of it as the beginning of a new fault collection cycle.